src/Mod/Draft/importSVG.py

```diff
@@ -382,7 +382,7 @@ def getsize(length, mode='discard', base=1):
         topx = {
             '': 1.0,  # default
             'px': 1.0,
-            'pt': 1.25,
+            'pt': 4/3,
```

Comment:
Although it is not necessary in Python 3, in Python 2 this integer division won't give you a floating point value. Therefore, I suggest to at least make one number have a decimal point, like `4./3`or `4.0/3` or `4.0/3.0`.

Again, this is only required if we want to maintain a bit of Python 2 compatibility.

Reply:
@vocx-fc : yep, wonder about that and check after the PR was submitted. This isn't a big job to keep compatibility here so better do it. I updated the PR (updated all lines for consistency even if not needed). Thanks for reviewing & reporting

Reply:
An even more elegant way is to add this line:
from __future__ import division

Then for Py2 4/3 also gives 1.3333333333333333 instead of 1